### DISCUSSION TITLE
Chartero

### DISCUSSION THREAD

**Discussion by simateimourian** [volatile-static/Chartero#138] [Q&A]
Hello everyone
I would be thankful if you helped me to use this plugin. I installed it but it does not work and noting shows in it . I categorized my article

**Comment by volatile-static** [comment]
Could you provide a screenshot?

**Comment by simateimourian** [comment]
Hi
I would be thankful to check it. The dashboard section is empty

**Comment by volatile-static** [comment]
I cannot see your **empty dashboard**.
If you are attaching an image, don't reply via email.
